Removes the incorrect assumption that the shebang must not have spaces,
since we correctly use "/usr/bin/env perl"

author: micketeer@gmail.com

--- configure.orig	2024-12-08 14:41:01.243852603 +0000
+++ configure	2024-12-08 14:41:51.837762787 +0000
@@ -3513,11 +3513,6 @@
  '')
    as_fn_error $? "perl not found" "$LINENO" 5
    ;;
- *' '* | *'	'*)
-  as_fn_error $? "The path to your Perl contains spaces or tabs.
-This would cause build failures later or unusable programs.
-Please use a path without spaces and try again." "$LINENO" 5
-  ;;
 esac
 
 # Save details about the selected perl interpreter in config.log.
